routes/index-routes.js

```diff
@@ -1,35 +0,0 @@
-const { compareSync } = require("bcrypt");
-const express = require("express");
-const Company = require("../models/company");
-const router = express.Router();
-const { averagesObject, getOverallReviewScore } = require("../public/javascripts/averages");
-
-// GET route for rendering the homepage. We get the companies from the DB and run the functions to calculate the averages.
-
-router.get("/", (req, res, next) => {
-  Company.find()
-    .populate("reviews")
-    .lean()
-    .then((companiesFromDb) => {
-      companiesFromDb.forEach((company, index) => {
-        if (company.reviews) {
-          company.averages = averagesObject(company.reviews);
-          company.overall = getOverallReviewScore(company.reviews);
-        }
-        company.index = index + 1;
-      });
-      const companiesWithReviews = companiesFromDb.filter((company) => company.overall !== undefined);
-      const companiesWithoutReviews = companiesFromDb.filter((company) => company.overall == undefined);
-      companiesWithReviews.sort((a, b) => {
-        return b.overall - a.overall;
-      });
-      const mergedArr = [...companiesWithReviews, ...companiesWithoutReviews];
-      mergedArr.forEach((company, index) => {
-        company.index = index + 1;
```

Comment:
Again here, I'm not sure what you're doing here but don't really like the look of it 😆